q2cli/tools.py

```diff
@@ -55,6 +70,9 @@ def export_data(path, output_dir):
               help='The format of the data to be imported. If not provided, '
                    'data must be in the format expected by the semantic type '
                    'provided via --type.')
+@click.option('--show-importable-types', is_flag=True, is_eager=True,
```

Comment:
@jairideout 

```$ python -m unittest q2cli.tests.test_cli.CliTests``` produces failures for every test.  

Could this be because this branch is behind upstream?  Should I rebase off of upstream?  Or is there a different way you guys run these unit tests?

You can see in my last commit that I started writing a test.   Am I on the right track here?  Is that something like what you're looking for?